pkg/util/utils_test.go

```diff
@@ -70,8 +70,8 @@ func TestValidateProxyURI(t *testing.T) {
 		"https://foo:b@r@test.com:454": true,
 	}
 	for uri, val := range urlList {
-		if ValidateProxyURI(uri) != val {
-			t.Fatalf("Expected '%t' Got '%t'", val, ValidateProxyURI(uri))
+		if ValidProxyURI(uri) != val {
```

Comment:
urlList does not cover enough failing cases. In the test all cases include port specification.